### DISCUSSION TITLE
Daily 04/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#170] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer miré algunos videos que tienen que ver con la parte del backed del proyecto, tratando de entender y ver como se pueden implementar las distintas alternativas. Empecé con los apuntes de la planning del sábado y estuve mejorando lo plasmado con anterioridad para facilitar la lectura. Por la tarde participé de la clase sincrónica del instituto.
Hoy terminé de pasar los apuntes y correcciones que ya se encuentran en el apartado de planning - review - retrospective de la wiki. Voy a volver a mirar algunos videos de backend y quiero empezar a trabajar en la parte de front que haya quedado pendientes. Por la noche voy a participar de la clase de inglés.

**Comment by BraianTroncoso** [comment]
Ayer estuve estudiando hasta tarde el tema del backend del proyecto, viendo diversas fuentes, también participe de la clase del ISPC. Hoy sigo con lo mismo trabajando en un código paralelo probando cosas.

**Comment by Valentino399** [comment]
Ayer estuve viendo y practicando el lenguaje de código java y hoy estaré viendo lo mismo

**Comment by MarielaGomez** [comment]
Ayer tuvimos clases, el profe explicó el crud que hay que realizar, aguardando suba apuntes, mientras estoy aprendiendo mediante tutoriales el crud de android con sqlite

**Comment by erick-lpz** [comment]
Ayer asistí a la clase para ver la conexión con la base de datos 
Hoy voy a leer el nuevo material de backend que dejaron en la plataforma del ISPC

**Comment by marceloedelgado** [comment]
Ayer estuve viendo como implementar el Recycler View para listar las tarjetas de los profesionales en un vertical scroll en la activity_catalogo . Me vino muy bien el intercambio y la aclaración del profesor Mariano cuando se lo consulté en clase

Hoy estuve viendo un poco de backend y base de datos en videos para en Android Studio
